test/test_heuristics.rb

```diff
@@ -802,6 +802,13 @@ def test_php_by_heuristics
     })
   end
 
+  def test_pkg_by_heuristics
+    assert_heuristics({
+      "DataFlex" => Dir.glob("#{fixtures_path}/Generic/pkg/DataFlex/*.pkg"),
```

Comment:
This needs to test your samples too, not just the fixtures contrived to match the regex.

```suggestion
      "DataFlex" => all_fixtures("DataFlex") + Dir.glob("#{fixtures_path}/Generic/pkg/DataFlex/*.pkg"),
```